code/modules/mob/living/carbon/xenomorph/castes/hivelord/castedatum_hivelord.dm

```diff
@@ -59,6 +59,7 @@
 		/datum/action/xeno_action/watch_xeno,
 		/datum/action/xeno_action/activable/psydrain,
 		/datum/action/xeno_action/activable/cocoon,
+		/datum/action/xeno_action/activable/Recycle,
```

Comment:
remove the captial R, keep the standard for xeno abilities to not have captial letters